src/test-harness/SparkEthereumTests.sol

```diff
@@ -529,50 +550,107 @@ abstract contract SparkEthereumTests is SparklendTests {
     }
 
     function _testRateTargetBaseIRMUpdate(
-        string memory symbol,
-        address       oldIRM,
-        address       newIRM,
-        uint256       oldSpread,
-        uint256       newSpread,
-        uint256       oldBaseRate,
-        uint256       newBaseRate
+        string                  memory symbol,
+        RateTargetBaseIRMParams memory oldParams,
+        RateTargetBaseIRMParams memory newParams
     )
         internal
     {
         SparkLendContext memory ctx = _getSparkLendContext();
 
         // Rate source should be the same
-        assertEq(IRateTargetBaseIRM(newIRM).RATE_SOURCE(), IRateTargetBaseIRM(oldIRM).RATE_SOURCE());
+        assertEq(ICustomIRM(newParams.irm).RATE_SOURCE(), ICustomIRM(oldParams.irm).RATE_SOURCE());
 
-        // Check spreads
-        assertEq(IRateTargetBaseIRM(oldIRM).getBaseVariableBorrowRateSpread(), oldSpread);
-        assertEq(IRateTargetBaseIRM(newIRM).getBaseVariableBorrowRateSpread(), newSpread);
```

Comment:
There should still be a check for the spread. This is not specifically checked in the general `_validateInterestRateStrategy`

Reply:
Added assertions